packages/expo/package.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "@knocklabs/expo",
+  "version": "0.3.1",
```

Comment:
Just to confirm, is this right for the initial version? Do we not want to start at 0.1.0?

Reply:
We can definitely start at v0.1.0. I wasn’t sure what’s preferable: restarting version numbering at 0 since this is a new package, or starting with the same version number as our RN SDK since the new package is essentially a fork.